doc/General/CODE_OF_CONDUCT.md

```diff
@@ -1,79 +0,0 @@
-source: General/CODE_OF_CONDUCT.md
```

Comment:
I don't think this should go should it? This file is linked to from the `CODE_OF_CONDUCT.md` file in the root.

Reply:
It felt a bit outdated, but mostly It was to de-clutter the menu

Reply:
Also, like mentioned above, it's from the fork period. Feels we're kinda past it.